### ISSUE TITLE
@ionic/app-scripts@2.1.4 depends on deprecated babili@0.1.2

### ISSUE THREAD

**Issue by kensodemann** [ionic-team/ionic-app-scripts#1230]
_From @Polyergic on September 19, 2017 12:58_

**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[X] **3.x**

**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

`npm install` gives this warning:

```
npm WARN deprecated babili@0.1.2: babili has been renamed to babel-minify. Please update to babel-minify
```

**Expected behavior:**

There should be no warnings

**Steps to reproduce:**

Put `@ionic/app-scripts` in `devDependencies` in `package.json`

**Other information:**

dependency found by `npm ls babili`:
```
└─┬ @ionic/app-scripts@2.1.4
  └── babili@0.1.2
```

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

Global packages are begging to shoot yourself in the foot, so I ran `npx ionic info`:
```

cli packages: (/Users/ssterling3/Documents/2017/IIT/Apps/Student Dashboard/mobile-dashboard/node_modules)

    @ionic/cli-utils  : 1.9.2
    ionic (Ionic CLI) : 3.9.2

global packages:

    Cordova CLI : 7.0.1

local packages:

    @ionic/app-scripts : 2.1.4
    Cordova Platforms  : browser 4.1.0
    Ionic Framework    : ionic-angular 3.6.0

System:

    Node  : v6.10.3
    npm   : 5.4.2
    OS    : macOS Sierra
    Xcode : Xcode 8.3.3 Build version 8E3004b
```

Node 6.10.3 is used to match the latest version [available in AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html)

_Copied from original issue: ionic-team/ionic#12928_

**Comment by kensodemann** [comment]
_From @danielsogl on September 19, 2017 13:37_

With the new release babili will be removed

**Comment by kensodemann** [comment]
_From @Polyergic on September 19, 2017 13:49_

We're currently devDepending on `"@ionic/app-scripts": "2.1.X"` (and generally basing our dependency versions on https://github.com/ionic-team/ionic2-app-base/blob/master/package.json).  

Will we have to update our dependencies to get the new release, or will it fall under "2.1.X"?

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

I am going to move this issue over to the `ionic-team/ionic-app-scripts` repo. Please follow the issue over there for updates.

Thank you for using Ionic

**Comment by danbucholtz** [comment]
We have deprecated Babili support.

Thanks,
Dan

**Comment by Polyergic** [comment]
Apparently not in the 2.1.X line.  My team is trying to avoid having to make ad-hoc updates to dependencies, so we can constrain the time spent on it and so we can more easily keep all of our projects and people on the same versions of everything we use.  We just updated our dependencies and started getting these warnings, having to update again right away defeats the purpose of our plan.  Also, if you're using semver, fixing this warning seems like it fits in the category where specifying 2.1.X ought to get the fix.

Our updated dependencies were based on https://github.com/ionic-team/ionic2-app-base/blob/master/package.json, where the app-scripts version changed only 11 days ago.  I know Angular has a release schedule, does Ionic have a release schedule as well?  It looks like we just happened to do our update right before you did yours, it would help if we could plan to do our updates right after you do yours.
